### ISSUE TITLE
Duplicate artifact exception

### ISSUE THREAD

**Issue by jschwartzentruber** [taskcluster/taskcluster#4904]
**Describe the bug**
```
[taskcluster:error] There was a conflict uploading artifact public/msys2.tar.bz2 - this suggests artifact public/msys2.tar.bz2 was already uploaded to this task with different content earlier on in this task.
```

**To Reproduce**

1. Create a windows/generic-worker task that creates an artifact
2. See error

**Expected behavior**
The artifact should have been created.

**Taskcluster version**
32.0.0 (generic worker version from task logs)

**Platform (please complete the following information):**

 - OS: windows
 - Version: server 2016

**Additional context**
The task is https://community-tc.services.mozilla.com/tasks/PD9I7UdHR7yoU56T5pPNuw

**Comment by djmitche** [comment]
I believe this is fixed in #4887, which hasn't been released (or deployed to community-tc) yet.

**Comment by jwhitlock** [comment]
PR #4887 was included in taskcluster v44.0.0, released to community-tc today.

**Comment by jwhitlock** [comment]
Please reopen if this issue occurs again.
